tools/build_config/build.ps1

```diff
@@ -25,6 +25,8 @@
   https://github.com/pace-neutrons/Herbert
 #>
 param (
+  # Run the Herbert build commands.
```

Comment:
This comment looks like a cut-and-paste from the next one. Did you mean "configure commands"?